letter-shop/index.md

```diff
@@ -0,0 +1,634 @@
+
+> In this simple example the payment is made using only a single transfer so the destination, and amount of the transfer, is the same as the destination and amount of the payment. You can see this from the fact that the same values are passed to the `sendTransfer()` function and used to build the ILP packet (`IlpPacket.serializeIlpPayment()`)
+
+> In a later tutorial we'll expand this example to pay across multiple networks and demonstrate how these values will often be different.
+
+Now that we've added the code to make our transfer let's run the script again and pay for our letter. This time you should see a message to say that we are making our payment, and waiting for the fulfillment.
+
+:tada: Congratulations, that's step 2 of 3 completed for our client! Now switch back to the terminal where you're running the shop and let's see if the payment has arrived...
+
+It hasn't, has it? So let's figure out why.
+
+### Finding your transfer on the ledger
+
+> The following short section is very specific to XRP Ledger but gives you an idea of what to do if you are debugging a failed payment. If you would rather skip ahead to the next section you can.
+
+Let's dissect what should have happened under the hood. You used the `pay.js` script to load the XRP Escrow plugin and prepare an escrow payment for 10 Drops (0.00001 XRP) to the address provided by the shop. 
+
+Our payment hasn't arrived at the shop so let's start by inspecting the XRP Testnet Ledger and see if that payment was prepared. We're going to use the REST API exposed by the same testnet server we connect our plugin to.
+
+First you need to find the sending account address you used. Look in `plugins.js` and find the account you configured your customer plugin with, or look at the output from running `pay.js` (you only need the Ripple address so trim off the ILP ledger prefix).
+
+The following will install a small tool that formats the JSON output to make it easier to read:
+
+```shell
+npm install -g jslint
+```
+
+Next use `curl` to POST an API call to the XRP Testnet server (replace the address with your sending address):
+
+```shell
+curl -X POST -d '{ "method": "account_objects", "params": [{"ledger_index":"validated", "account": "YOUR-SENDING-ADDRESS", "type": "escrow"}]}' https://client.altnet.rippletest.net:51234
+```
+
+*Installing `curl` is out of scope for this tutorial so if you don't already have it you can skip ahead or figure that bit out yourself.*
+
+You should get back a JSON object that shows all the transactions sent from your account. In among them should be the transaction you just created. 
+
+So our investigation has revealed that the transfer has been created on the ledger. We need to go back to our shop service and figure out why it wasn't received.
```

Comment:
Nice! I like how this shows that something really happened on the XRP testnet ledger, and also how in the tutorial you get one step further each time.